src/main/scala/mesosphere/marathon/tasks/OfferMatcher.scala

```diff
@@ -205,8 +214,15 @@ class IterativeOfferMatcher @Inject() (
         launchedTasks += tasks.size()
       }
       else {
-        log.debug("declining offer {}", offer.getId)
-        driver.declineOffer(offer.getId)
+        iterativeOfferMatcherConfig.rejectOfferDuration.get match {
```

Comment:
I think we need to remove the `.get`. Looks like the type of `rejectOfferDuration` is `Option[Long]`.